### PR TITLE
RFC: systemd: add version 231

### PR DESCRIPTION
**Pull request by sknsean**
Signed-off-by: Sean Nyekjaer sean.nyekjaer@prevas.dk

### CODE REVIEW

**Comment by sknsean** [comment]
Depends on https://github.com/oe-lite/base/pull/392 https://github.com/oe-lite/base/pull/390 https://github.com/oe-lite/base/pull/389 https://github.com/oe-lite/core/pull/169

File: recipes/systemd/systemd.inc [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+		--enable-timedated \
+		--enable-timesyncd \
+		--enable-kdbus \
+		--enable-logind \
+		--enable-sysusers \
+		--enable-bzip2 --enable-lz4 --enable-zlib \
+                "
+#--enable-xz
+
+DEPENDS += "libpthread libcap util-linux-libmount libdl libm librt libresolv libcrypt native:coreutils-ln \
```
Comment (esben) [inline recipes/systemd/systemd.inc]: Why did you add native:coreutils-ln ?

I don't think we ever added dependencies against standard host utils like that before...

Comment (esben) [inline recipes/systemd/systemd.inc]: Please depend on libmount instead of util-linux-libmount.

File: recipes/systemd/systemd.inc [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+		dbus-libdbus-1 zlib bzip2-libbz2 lz4-liblz4"
```
Comment (esben) [inline recipes/systemd/systemd.inc]: Please depend on the proper library provides like libdbus-1, libz, libbzip2, liblz4 instead of concrete package names.

File: recipes/systemd/systemd.inc [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+		/usr/lib/sysctl.d/ \
+		/usr/lib/kernel/install.d/ \
+		/usr/share/factory/etc/ \
+		/usr/share/dbus-1/ \
+		/usr/lib/rpm/macros.d/ \
+		/usr/share/polkit-1/ \
+		"
+FILES_${PN}-udev += "./etc/udev \
+		/usr/bin/udevadm \
+		/usr/lib/libudev.so* \
```
Comment (sknsean) [inline recipes/systemd/systemd.inc]: Should I create an ${PN}-udev and a ${PN}-libudev ?

File: recipes/systemd/systemd.inc [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+		${libdir}/sysctl.d/ \
+		${libdir}/kernel/install.d/ \
+		${datadir}/factory/etc/ \
+		${datadir}/dbus-1/ \
+		${libdir}/rpm/macros.d/ \
+		${datadir}/polkit-1/ \
+		"
+FILES_${PN}-udev += "./etc/udev \
+		${bindir}/udevadm \
+		${libdir}/libudev.so* \
```
Comment (kimrhh) [inline recipes/systemd/systemd.inc]: how about adding auto-package stuff so we get it split up correctly?
